steps/query/spec.schema.json

```diff
@@ -0,0 +1,62 @@
+
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "servicenow": {
+      "type": "object",
+      "description": "A mapping of ServiceNow account configuration.",
+      "properties": {
+        "connection": {
+          "type": "object",
+          "x-relay-connectionType": "servicenow",
+          "description": "A Relay ServiceNow connection to use",
+          "properties": {
+            "host": {
+              "type": "string",
+              "description": "ServiceNow instance host"
+            },
+            "secretAccessKey": {
+              "type": "string",
+              "description": "ServiceNow user"
+            },
+            "password": {
+              "type": "string",
+              "description": "ServiceNow password"
+            }
+          },
+          "required": [
+            "host", 
+            "user",
+            "password"
+          ]
+        }
+      },
+      "required": [
+        "connection",
+        "resource"
+      ]
+    },
+    "resource": {
+      "type": "string",
+      "description": "Resource to query"
+    },
+    "query": {
+      "type": "string",
+      "description": "Query to run"
+    },
+    "arguments": {
+      "type": "object",
+      "description": "Arguments to use"
+    },
+    "filter": {
+      "type": "string",
+      "description": "Optional Python lambda to filter data"
```

Comment:
Can we mark this as experimental/unstable? Once we fix the outputs limit I'd like to remove it and offload to somewhere else (ideally the expression language).